conda-forge.yml

```diff
@@ -1,3 +1,10 @@
+# Uncomment for Windows re-rendering only.
+#
+#matrix:
+#  - [[VC_VERSION, 9], [CONDA_PY, 27]]
+#  - [[VC_VERSION, 10], [CONDA_PY, 34]]
+#  - [[VC_VERSION, 14], [CONDA_PY, 35]]
```

Comment:
Basically my suggestion until we have something is re-render with this commented and commit all changes. Then uncomment this, re-render, and only commit `appveyor.yml` (possibly amending the last re-rendering commit). That should get us a little closer to a stable process for doing this.